qcengine/programs/mp2d.py

```diff
@@ -0,0 +1,314 @@
+"""Compute dispersion correction using Greenwell & Beran's MP2D executable."""
+
+import copy
+import os
+import pprint
+import re
+import sys
+import traceback
+from decimal import Decimal
+
+import numpy as np
+import qcelemental as qcel
+from qcelemental.models import FailedOperation, Result
+
+from .dftd3 import dashparam
+from .dftd3.runner import module_driver  # nasty but temporary and better than duplicating fn
+from ..util import which
+from .executor import ProgramExecutor
+from ..extras import provenance_stamp
+
+pp = pprint.PrettyPrinter(width=120, compact=True, indent=1)
+
+
+class MP2DExecutor(ProgramExecutor):
+
+    _defaults = {
+        "name": "MP2D",
+        "scratch": True,
+        "thread_safe": True,
+        "thread_parallel": False,
+        "node_parallel": False,
+        "managed_memory": False,
+    }
+
+    class Config(ProgramExecutor.Config):
+        pass
+
+    def __init__(self, **kwargs):
+        super().__init__(**{**self._defaults, **kwargs})
+
+    @staticmethod
+    def found(raise_error=False) -> bool:
```

Comment:
Any reason for `raise_error` here. This doesn't associate with other methods it seems.